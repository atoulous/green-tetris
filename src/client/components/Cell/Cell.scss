.grid .row .cell {
    border: solid gray 1px;
    display: flex;
    border-radius: 0;
    margin: 0;
    width: 30px;
    height: 30px;
}

.spectrum .row .cell {
    border: solid gray 1px;
    width: 100%;
    flex: 1;
}

.nextPiece .row .cell {
    width: 15px !important;
    height: 15px !important;
}