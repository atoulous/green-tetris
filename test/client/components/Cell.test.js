import React from 'react';
import renderer from 'react-test-renderer';

import Cell from '../../../src/client/components/Cell/Cell';

it('cell with fill', () => {
  const tree = renderer
    .create(<Cell cell={{ fill: true, color: 'cyan' }} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('color with no fill', () => {
  const tree = renderer
    .create(<Cell cell={{ fill: false, color: 'cyan' }} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
