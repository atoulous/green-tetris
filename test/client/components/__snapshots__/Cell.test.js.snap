// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cell with fill 1`] = `
<div
  className="cell"
  style={Object {}}
/>
`;

exports[`color with no fill 1`] = `
<div
  className="cell"
  style={Object {}}
/>
`;
